proto/attest.proto

```diff
@@ -29,7 +29,7 @@ message Attestation {
   bytes event_log = 3;
   // Optional information about a GCE instance, unused outside of GCE
   GCEInstanceInfo instance_info = 4;
-  // A TCG Canonical Event Log.
+  // A TCG Canonical Event Log measured to registers (PCR/RTMR).
```

Comment:
I think this update is unnecessary. Really, it should say "A COS event log using the TCG Canonical Event Log format"

Reply:
done. I was thinking to make it clear this can be an eventlog based on PCR or RTMR